server/app/Models/Doc.php

```diff
@@ -230,67 +220,33 @@ public function getPagesAttribute()
         return $this->getPages();
     }
 
-    public function getCommentCount()
-    {
-        return $this->comments()->count();
-    }
-
     public function getCommentCountAttribute()
     {
-        return $this->getCommentCount();
-    }
-
-    public function getAnnotationCount()
-    {
-        return $this->annotations()->count();
-    }
-
-    public function getAnnotationCountAttribute()
-    {
-        return $this->getAnnotationCount();
+        // note that this is only the top level comment count
```

Comment:
Some client tests are failing because it's expecting there to be a count of all sub comments.